ocs_ci/ocs/ui/views.py

```diff
@@ -6,11 +6,14 @@
 ODF_OPERATOR = "OpenShift Data Foundation"
 
 login = {
+    "pre_login_page_title": "Log In",
+    "login_page_title": "Log in · Red Hat OpenShift Container Platform",
```

Comment:
Locator type is missing.

Reply:
Not sure that I understood the problem as I wanted to store a string only and not a tuple of locator that includes By.SOMETHING. 
For example: [this line](https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/ocs/ui/views.py#L9)
Should I move those to a constants file ?

Reply:
Got it, it's okay to keep it here as well. Not sure if we have a better alternative.